src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/AuthorizationFunctionalTest.java

```diff
@@ -332,6 +333,73 @@ public void responseValidateV2(Map<String, Object> orgResponse) {
 
     }
 
+    public void validateAccessTypesInRetrievedUser(Map<String, Object> searchResponse, String expectedStatus,
+                                                   Boolean rolesReturned) {
+        assertThat(searchResponse.get("users")).asList().isNotEmpty();
+        assertThat(searchResponse.get("organisationIdentifier")).isNotNull();
+        assertThat(searchResponse.get("organisationProfileIds")).isNotNull();
+        List<HashMap> professionalUsersResponses = (List<HashMap>) searchResponse.get("users");
```

Comment:
check size of professionalUsersResponses is 1

Reply:
Thank you for review. Its fixed now. please review.